src/rev_ai/models/insights/topic_extraction/topic_extraction_result.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Topic extraction result model"""
+
```

Comment:
sometime it is a [single line break](https://github.com/revdotcom/revai-python-sdk/pull/79/files#diff-6cb75734fa52c453b1067020e064b4a351b5dd91bbe1177d2e2be677357d1f63R3), sometimes double

Reply:
It is a single line break between top of file and imports and a double between top of file and class declarations. As far as I can tell this is consistent